src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="bar"
+      data-qa="big"
+    >
+      <label for="search-1"></label>
+      <input
+        type="search"
+        name="search-1"
+        id="search-1"
+        class="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+    <form
+      action="#"
+      method="get"
+      class="bar"
+      data-qa="small"
+    >
+      <label for="search-2"></label>
+      <input
+        type="search"
+        name="search-2"
+        id="search-2"
+        class="search small-class"
```

Comment:
it's better to give this class the name of `search--small`, but you also need to have `search--big`.
at the moment you are styling everything under the `search` class and this does not make a lot of sense. for a better understanding of what is happening, I suggest you have a class `search` and both search bars have this class. it contains the common styles. and you also have `search--small` and `search--big` which contain classes that differ. i think it will be a lot more structured